src/http/cookie.cr

```diff
@@ -192,6 +192,16 @@ module HTTP
       end
     end
 
+    # tell the browser to delete the cookie.
+    # Set its value to an empty string
+    # and set its expiration time to the past
+    # Browsers delete cookies that have expired.
+    def destroy
+      self.value = ""
+      self.expires = Time::UNIX_EPOCH
+      self.max_age = Time::Span.zero
+    end
```

Comment:
```suggestion
    # Tells the browser to expire the cookie by setting its 
    # max age time to zero and its expiration time to the past.
    def expire
      self.expires = Time::UNIX_EPOCH
      self.max_age = Time::Span.zero
    end
```

Reply:
Could you elaborate on why you suggest that we remove setting the value to a blank string?

Reply:
Sure, my reasoning was twofold:

1. `expire` method name IMO implies only cookie expiration, not emptying its value
2. Spec doesn't mention (or I've not seen it does) clearing out the value as the requirement for a cookie expiration

Reply:
I believe destroy will better represent what the function does. And there are many methods with a name similar to `#expire` I think calling it `#destroy` would be more clear.

Reply:
Clearing the value is not a requirement for expiration. But I think it makes a lot of sense *not* to send the value you want gone to the client again.